app/test_main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+import app.main
+from app.main import can_access_google_page
+from unittest import mock
+
+
+def test_access_googl_page_without_internet() -> None:
```

Comment:
There's a typo in the function name: 'test_access_googl_page_without_internet'. It should be 'test_access_google_page_without_internet'.